bindata/network/ovn-kubernetes/ipsec.yaml

```diff
@@ -1,4 +1,3 @@
-{{if .OVNIPsecDaemonsetEnable}}
```

Comment:
Suggestion: add a log line to the console output in the container that indicates whether ipsec is enabled. I know it has no effect here but I foresee some curious / nervous administrators wondering why their nodes are suddenly running ipsec.

Also, the LivenessProbe is probably a bad idea... it will kill -9 the container if it fails, which is rarely desired. Every LivenessProbe I've ever added turned out to be a bad mistake.

Reply:
> Suggestion: add a log line to the console output in the container that indicates whether ipsec is enabled. I know it has no effect here but I foresee some curious / nervous administrators wondering why their nodes are suddenly running ipsec.
> 

This is my only worry really. "Oh no, I have accidently configured IPsec". I guess the log would help.

> Also, the LivenessProbe is probably a bad idea... it will kill -9 the container if it fails, which is rarely desired. Every LivenessProbe I've ever added turned out to be a bad mistake.

The reason that is here is because we there was a random segmentation fault at initialization (due to a bug in libreswan) that would cause libreswan to die, this would catch this failure and restart the pod (and libreswan). I don't know if the libreswan change has propagated yet (but I haven't seen the segfault in the CI in a while) - s, I anm a bit reluctant to remove it right now.

Reply:
Aha. That seems like one of the few cases when a LivenessProbe is actually useful.

Reply:
Yeah, I would potentially like to remove it in the future. I think RHEL9 is when I am sure that fix has been applied and even then, I would spend some time going through the CI logs to see if I saw it show up again.